index.html

```diff
@@ -263,5 +263,25 @@ <h1 class="first-title2">About Myself</h1>
         </div>
     </div>
 </section>
+<footer class="footer" id="contact">
+    <div class="container-3">
+        <div class="parent">
+            <img src="./Images/contactformshapes-mobile.png" class="over-img" alt="contactformimage"/>
+        </div>  
+    <h1 class="first-title3">Contact me</h1>
+    <p class="description6">
+		If you have an application you are interested in developing,a feature that you need built or a project that needs coding. I’d love to help with it
+    </p>
+    <form class="contactform" action="https://formspree.io/f/myyoqzdn" method="POST">
+		<input type="text" name="name" placeholder="Full Name" maxlength="30" required />
+		<input type="email" name="email" placeholder="Email Address" required />
+		<textarea name="message" placeholder="Write your message here..." maxlength="500" required > </textarea>
```

Comment:
- [ ] The required validation for the textarea doen't seem to work. Kindly check it.